git-functions.sh

```diff
@@ -189,4 +190,16 @@ function gpof {
     echo "Yeah, better so!"
   fi
   
+}
+
+function gpuf {
+  echo "Are you sure you want to do a forced push? (y,N)"
```

Comment:
```suggestion
  echo "Are you really sure you want to do a forced push upstream? (y,N)"
```

Added some emphasis and `upstream` so it's different from the warning in `gpo`.